flags.go

```diff
@@ -573,3 +575,20 @@ func (oi *OctalInt) Set(value string) (err error) {
 func (oi OctalInt) String() string {
 	return fmt.Sprintf("%o", oi)
 }
+
+func metadataCacheTTL(statCacheTTL, typeCacheTTL time.Duration, ttlInSeconds int64) (metadataCacheTTL time.Duration) {
```

Comment:
since this is a helper that is used at the time of mounting, so I would prefer keeping it in https://github.com/GoogleCloudPlatform/gcsfuse/blob/read_cache_release/internal/mount/flag.go